WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -497,7 +499,30 @@ public void setPlanID(long planID) {
     public String getPlanShortName() {
         return StringUtils.notNullStr(planShortName);
     }
+
     public void setPlanShortName(String name) {
         this.planShortName = StringUtils.notNullStr(name);
     }
+
+    public String getCapabilities() {
+        return capabilities;
+    }
+
+    public void setCapabilities(String capabilities) {
+        this.capabilities = capabilities;
+    }
+
+    public boolean checkCapability(Capability capability) {
+        // If a capability is missing it means the user don't have it.
+        if (capabilities == null || capabilities.isEmpty() || capability == null) {
+            return false;
+        }
+        try {
+            JSONObject jsonObject = new JSONObject(capabilities);
```

Comment:
The capabilities for now is just for checking them for certain People Management actions. I don't think there will be anyway to update capabilities without updating someone's role. They are tied to each other.

I'll make the change suggested by @hypest in another PR before this gets merged into `develop`. Thanks for the feedback @maxme!